plugin/evm/block.go

```diff
@@ -101,15 +102,53 @@ func (b *Block) syntacticVerify() error {
 }
 
 // Verify implements the snowman.Block interface
+// Since VerifyWithContext is implemented, we never expect this to be called.
```

Comment:
@StephenButtolph we expect this to be called prior to the proposervm activating right?

```suggestion
// Since VerifyWithContext is implemented, Verify should not be called after the proposervm activates
```

Reply:
AFAIK, `Verify` will still be called even after the proposervm activation if `ShouldVerifyWithContext` returns false for a given block (which will be the case for blocks that don't depend on any context). 

This allows optimizations because the verification result is known to never change for a block if it does not depend on a context, but that is not the case for blocks that do depend on a context because the verification result may change if the context changes.